custom_components/localtuya/pytuya/__init__.py

```diff
@@ -551,7 +551,14 @@ class TuyaProtocol(asyncio.Protocol, ContextualLogger):
     """Implementation of the Tuya protocol."""
 
     def __init__(
-        self, dev_id, local_key, protocol_version, enable_debug, on_connected, listener
+        self,
+        dev_id,
+        local_key,
+        node_id,
+        protocol_version,
+        enable_debug,
+        on_connected,
+        listener,
```

Comment:
Trailing ','

Reply:
on listener? seems visual studio code likes putting it there. I don't understand the difference.